client/src/Login.js

```diff
@@ -13,15 +13,15 @@ class Login extends React.Component {
     this.handleChange = this.handleChange.bind(this)
   }
 
-  handleSubmit (e) {
+  handleSubmit(e) {
     // Implement check for duplicate users
```

Comment:
[nit] Add a `TODO:` in front of the comment, so it's not misinterpreted as describing the if statement below